### DISCUSSION TITLE
Red dot.

### DISCUSSION THREAD

**Discussion by avalon60** [Akascape/CTkMessagebox#23] [Q&A] [answered]
Is there a way to remove the Mac-like, red dot from the top-right of the pop-up, for non Mac implementations? It seems like something you would only expect to see on a Mac. Or is this just meant  to help grab the attention?

![image](https://github.com/Akascape/CTkMessagebox/assets/89534395/0c1f7c91-4825-4047-acb0-ed56c09953c1)

**Comment by Akascape** [comment, marked as answer]
@avalon60 We have a cross mark button also.
![Screenshot](https://github.com/Akascape/CTkMessagebox/assets/89206401/6d92106e-27c7-4ccd-b16b-06a48d2b62f5)

Use `cancel_button="cross"` in the messagebox parameters.

**Reply by avalon60** [reply]
Would it be possible to enhance it so that the default is based on O/S? That is for Windows and Linux the default is a cross and for Mac the circle?

**Comment by ordec2** [comment]
yes
